include/secp256k1_musig.h

```diff
@@ -366,13 +366,18 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_musig_partial_sig_verif
  *  Out:          sig: complete signature (cannot be NULL)
  *  In:  partial_sigs: array of partial signatures to combine (cannot be NULL)
  *             n_sigs: number of signatures in the partial_sigs array
+ *            tweak32: if `combined_pk` was tweaked with `tweak_add` after
```

Comment:
nits:
 * `tweak_add` is not the full function name
 * s/if/If

Reply:
why capital letter for `if`? That's inconsistent with the rest.

Reply:
It's a full sentence. But I'm not going to start an argument about this. :man_shrugging: